cms/static/cms/sass/components/_structureboard.scss

```diff
@@ -234,11 +234,11 @@
             overflow: hidden;
```

Comment:
In line 188, I believe this should be:  `padding-inline-start: $structure-draggable-inner-padding`